### PR TITLE
Add location category and description fields to the location model

### PR DESCRIPTION
**Pull request by cmplx-xyttmt**
### What does this PR do?
Adds a location category to the location model. Also adds the following extra properties based on the category field:
- **Location description**: a description of the surrounding areas e.g residential, hospital e.t.c
- **Night limit**: The DB threshold at which an area is considered noisy at night.
- **Day limit**: The DB threshold at which an area is considered noisy during the day.

The above are based on the [NEMA Noise Standards and Control Regulations](http://nema.go.ug/sites/all/themes/nema/docs/noise_standards_and_control_regulations.pdf)

### How to test this PR?
- Run the application.
- Go to the device page. Confirm that the location info is shown.
- Confirm that the category can be edited.
- Make a call to the endpoint `http://localhost:8000/devices/locations` and confirm that the fields `location_description`, `day_limit` and `night_limit` are present in the response.

Example API response:
```
[
    {
        "id": "276c67f1-5302-4ce5-996d-e152c5e6249f",
        "latitude": 0.335984,
        "longitude": 32.57291,
        "city": "Kampala",
        "division": "Central Division",
        "parish": "Wandegeya",
        "village": "N/A",
        "latest_metric": {
                .....
        },
        "location_description": "Category A: hospital, convalescence home, sanatorium, home for the aged and higher learning institute, conference rooms, public library, environmental or recreational sites",
        "day_limit": 45,
        "night_limit": 35
    }
]
```

### CODE REVIEW

File: devices/serializers.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Meta:
         model = Location
         fields = [
             'id', 'latitude', 'longitude', 'city',
-            'division', 'parish', 'village', 'latest_metric'
+            'division', 'parish', 'village', 'latest_metric', 'location_description', 'day_limit', 'night_limit',
```
Comment (NLSanyu) [inline devices/serializers.py:22]: @cmplx-xyttmt I'm getting an out of range error (`IndexError: list index out of range`), coming from the `latest_metric` method. 
Screenshot 2022-03-18 at 13 34 33
I'm guessing this happens for the devices that I have in my local instance that have no metrics yet.

Comment (NLSanyu) [reply devices/serializers.py:22]: > @cmplx-xyttmt I'm getting an out of range error (`IndexError: list index out of range`), coming from the `latest_metric` method.
> 
> Screenshot 2022-03-18 at 13 34 33
> 
> I'm guessing this happens for the devices that I have in my local instance that have no metrics yet.

When I upload at least one metric for every device in my local db, it works perfectly:

Screenshot 2022-03-18 at 13 44 26

So, it's just a edge case for when a device has no metrics yet

Comment (cmplx-xyttmt) [reply devices/serializers.py:22]: Since this is unrelated to this PR, I'll just create a new task for it and fix later

Comment (NLSanyu) [reply devices/serializers.py:22]: Alright

**Review by NLSanyu — APPROVED** [review]

**Pull request merged**